corehq/apps/ota/models.py

```diff
@@ -68,7 +69,7 @@ def _write_restore_blob(self, restore, db):
         elif isinstance(restore, bytes):
             restore = StringIO(restore)
 
-        info = db.put(restore)
+        info = db.put(restore, random_url_id(16))
```

Comment:
👍 on finding and fixing this actually-kind-of-serious collision issue for restores!